.travis.yml

```diff
@@ -17,9 +17,15 @@ addons:
   apt:
     packages:
       - oracle-java8-installer
+  postgresql: "9.4"
+
+services:
+  - postgresql
 
 before_script:
-- export GIT_BRANCH=$TRAVIS_BRANCH
+  - export GIT_BRANCH=$TRAVIS_BRANCH
+  - psql -c "create user test password 'test';" -U postgres
```

Comment:
We were not able to configure the database using /config/database.yml.travis, so it has been removed. A _handmade_ script has been added on the before_script section on .travis.yml